public/app/features/alerting/unified/MoreActionsRuleButtons.tsx

```diff
@@ -40,7 +42,13 @@ export function MoreActionsRuleButtons({}: Props) {
 
   if (canExportRules) {
     menuItems.push(
-      <MenuItem label="Export all Grafana-managed rules" key="export-all-rules" onClick={toggleShowExportDrawer} />
+      <MenuItem
+        label="Export all Grafana-managed rules"
+        key="export-all-rules"
+        onClick={toggleShowExportDrawer}
+        disabled={!enableExport}
```

Comment:
i think it would be more clear using `disableExport` defaulting to false,  instead of `enableExport` as a property, wdyt?

Reply:
Well, I think yes and no. I generally try to avoid negative forms for variable names as they tend to be more difficult to reason about (they lead to double negations `!disabled`)
While 'disableExport' may seem more suitable in this instance, I'm concerned it could pose issues in the long term